src/hooks/useGetUserPlayStreak.ts

```diff
@@ -2,25 +2,33 @@ import { getGetUserPlayStreakQueryKey } from '@/helpers/getQueryKeys'
 import { UserPlayStreak } from '@hyperplay/utils'
 import { useQuery, useQueryClient } from '@tanstack/react-query'
 
-export function useGetUserPlayStreak(
+export function getGetUserPlaystreakQuery(
   questId: number | null,
   getUserPlayStreak: (questId: number) => Promise<UserPlayStreak>
 ) {
-  const queryClient = useQueryClient()
   const queryKey = getGetUserPlayStreakQueryKey(questId)
-  const query = useQuery({
+  return {
     queryKey,
     queryFn: async () => {
       if (questId === null) {
         return null
       }
       const response = await getUserPlayStreak(questId)
       if (!response) return null
-      return response
+      return { questId: questId, userPlayStreak: response }
```

Comment:
the user playstreak response does not have the quest id, but I need it for the `useQueries` call in https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/pull/1124/files#diff-191d2dad24a032e65214284f9b14a96e4f1445c55a177f60c677a169185db5c3 `src/frontend/hooks/useGetQuestStates.ts`